src/build-disk-image.ts

```diff
@@ -21,10 +21,10 @@ import * as extensionApi from '@podman-desktop/api';
 import * as os from 'node:os';
 import * as fs from 'node:fs';
 import { resolve } from 'node:path';
+import * as containerUtils from './container-utils';
 
 const bootcImageBuilderContainerName = '-bootc-image-builder';
-const bootcImageBuilderName = 'quay.io/centos-bootc/bootc-image-builder:latest-1704948606';
-let diskImageBuildingName: string;
+const bootcImageBuilderName = 'quay.io/centos-bootc/bootc-image-builder:latest';
```

Comment:
I don't think I'm comfortable with this yet - changing one argument could easily break our extension and they're likely not trying to maintain compatibility yet. Can we stick with a tagged version or sha, at least for now?

Reply:
you might pin the release to a given sha but add an option that will be in preferences where as a user you might/could override or a checkbox 'dev/stable' or drop-down 'latest release/ dev release or extension's versions

So default is locked but you allow to test other versions